### ISSUE TITLE
[Protection Paladin] Judgment of Light (Tier 6 talent)

### ISSUE THREAD

**Issue by Tolno** [Valhalla-Gaming/Tracker#7677]
**Describe the issue you're having**: This talent (listed as "working", btw) actually heals the ennemy...

**Explain how you expect it work**: Well, it should heal me and every allied attacker of any target with this debuff applied.

**Steps to reproduce the problem**:

1. Take the talent
2. Judge ennemy target and hit him
3. Look at the nice green text over HIM, not over you, and open your Combat Log, and see all these nice heals given to your ennemy :)

**Links to Wowhead, YouTube, etc**: [http://www.wowhead.com/spell=183778/judgment-of-light](url)

**Comment by Moustachos** [comment]
Can't reproduce.

Do you see Judgment of Light's debuff on your target when you use Judgment?

**Comment by tribaljet** [comment]
Yes, the Judgment of Light debuff is applied onto enemy targets and does heal targets accordingly.
